src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,42 @@
+.Navbar {
+  width: 90%;
+  height: 50px;
+  margin: 0;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--color-white);
+  
+  &__left {
+    display: flex;
+    justify-content: flex-start;
+
+    .Navbar__icon img {
+      width: 150px;
+    }
+  }
+  
+  &__right {
+    flex: 3;
+    gap: 2em;
+    display: flex;
+    justify-content: flex-end;
+    
+    .Navbar__links {
+      font-family: var(--inter);
+      color: var(--color-bunting);
+      text-decoration: none;
+      font-size: 14px;
+      line-height: 14px;
+    }
+    
+    .Navbar__btn {
+      padding: 16px;
+      color: var(--color-blue-ribbon);
+      border-radius: 8px;
+      outline: 2px solid;
+      outline-color: var(--color-blue-ribbon);
+      font-family: var(--inter);
```

Comment:
I would remove outline (both of them) and use 
border: 1px solid var(--color-blue-ribbon)